test/testrunner.py

```diff
@@ -97,8 +97,10 @@ def main():
     all_image_list = dlc_images.split(" ")
     standard_images_list = [image_uri for image_uri in all_image_list if "example" not in image_uri]
     eks_cluster_name = None
-    benchmark_mode = "benchmark" in test_type
-    specific_test_type = re.sub("benchmark-", "", test_type) if benchmark_mode else test_type
+    # benchmark_mode = "benchmark" in test_type
```

Comment:
Config test to have benchmark script tested under PR context. Will be reverted before merging in